helpers/englishUlbHelpers.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * ugntHelpers.js - this is the code called by ugntParse.js to download and convert greek resources (was BHP and now
+ *                  UGNT.
+ */
+
+import path from 'path-extra';
+import * as bible from '../scripts/bible';
+import * as UsfmParseHelpers from './usfmParseHelpers';
+
+//TODO: EN_ULB_URL path is a temporary repo, needs to be updated to final location.
+
+const EN_ULB_URL = 'https://git.door43.org/photonomad0/AlignedUlb_en/raw/master';
```

Comment:
@PhotoNomad0 Isn't the project supposed to come from the catalog?

Reply:
As marked in the TODO, this is temporary until the ULB has been aligned.